hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestJdbcSource.java

```diff
@@ -73,12 +73,12 @@ public static void beforeAll() throws Exception {
   @BeforeEach
   public void setup() throws Exception {
     super.setup();
-    PROPS.setProperty("hoodie.deltastreamer.jdbc.url", "jdbc:h2:mem:test_mem");
-    PROPS.setProperty("hoodie.deltastreamer.jdbc.driver.class", "org.h2.Driver");
-    PROPS.setProperty("hoodie.deltastreamer.jdbc.user", "test");
-    PROPS.setProperty("hoodie.deltastreamer.jdbc.password", "jdbc");
-    PROPS.setProperty("hoodie.deltastreamer.jdbc.table.name", "triprec");
-    connection = DriverManager.getConnection("jdbc:h2:mem:test_mem", "test", "jdbc");
+    PROPS.setProperty("hoodie.streamer.jdbc.url", "jdbc:h2:mem:test_mem");
+    PROPS.setProperty("hoodie.streamer.jdbc.driver.class", "org.h2.Driver");
+    PROPS.setProperty("hoodie.streamer.jdbc.user", "sa");
+    PROPS.setProperty("hoodie.streamer.jdbc.password", "");
+    PROPS.setProperty("hoodie.streamer.jdbc.table.name", "triprec");
+    connection = DriverManager.getConnection("jdbc:h2:mem:test_mem", "sa", "");
```

Comment:
Similar here for the JDBC user name and password.

Reply:
reverted, will do it in separate PR.